### ISSUE TITLE
Remove the printing of the type of the exception from the CLI

### ISSUE THREAD

**Issue by oritstone** [openscore/score-language#42]
Currently, there is also usually a duplication of the exception type printed there...
I don't think it should be there at all since in almost all of the cases we have a RuntimeException, and the type adds no incentive for the problem.

**Comment by orius123** [comment]
:+1:

**Comment by meirwah** [comment]
There are cases where it is SlangException, and then it is not duplicate.. i think we wil prospond the fix of this, as this is not urgent...

**Comment by oritstone** [comment]
I don't think that the Exception type is interesting at all, since in 90% of the cases we wrap it with RuntimeException anyway
If someone is interested, he can always address the log

**Comment by Bonczidai** [comment]
currently the displayed error message has the following format:
```console
Command failed java.lang.RuntimeException: Failed compilation for file : http_client_get.sl ,Exception is : java.lang.Ru
ntimeException: For operation 'http_client_action' syntax is illegal.
key: naughty_modifier in input: httpClientCookieSession is not a known property
```

@CimpeanRadu @tethryus WDYT about this issue?
